lib/metadata/mongoclient/backend.js

```diff
@@ -334,7 +358,11 @@ class MongoClientInterface {
             value: objVal,
         }, {
             upsert: true,
-        }, () => cb());
+        }, err => {
+            log.error('putObjectNoVer: error putting obect with no versioning',
```

Comment:
This line is causing error ""Cannot read property 'message' of null", needs an if (err) statement